src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -178,10 +183,10 @@ private set
         _previewChanges = previewChanges;
 
         _initialRenameText = triggerSpan.GetText();
-        this.ReplacementText = _initialRenameText;
```

Comment:
could you undo these?  `this.` allows us to see that these are accessing members on the instance.

Reply:
whoops I think accidentally some sort of code cleanup must have ran